radvd64: introduce RADVD64 container

mike-scott opened the pull request:
This is a specialized deployment of radvd for use with BLE 6lowpan and
802.15.4 networks.

Included is an interface up trigger which assigns an IPv6 prefix to the
6lowpan interface as it goes up and down when nodes are connected and
disconnected.

Signed-off-by: Michael Scott <mike@foundries.io>

mike-scott commented:
@ricardosalveti this container now does the interface up trigger to add the IPv6 prefix that we need.  Once integrated into our docker compose, we can drop the bt0 trigger from the LmP build.

Note: this container can be re-used multiple times for each interface where radvd is needed.  IE: bt0, lowpan0, etc. as long as each has different parameters for interface and ipv6 prefix.

mike-scott commented:
Ugh.. it seems like the current interface trigger logic isn't working.  Need to make a tweak to the script.

mike-scott commented:
@ricardosalveti I'm not married to the way I'm doing the interface up detection.  If you have a better idea, let me know.

doanac commented on radvd64/Dockerfile:
```diff
@@ -0,0 +1,30 @@
+RUN chmod +x interface-monitor.sh
+
+COPY start.sh start.sh
+RUN chmod +x start.sh
```
Isn't there something about docker images being a little more efficient if you put all the "RUN" commands into a single pass? Also - if you just add the files with +x, then you don't have to bother with the chmod stuff.

mike-scott replied:
Fixed this

mike-scott commented:
- Updated logic in the interface monitor script so that it works (a bit janky still)
- Fixed dbus mounting w/ var/run as tmpfs

Tested this and it's working reliably now
